### PR TITLE
Add message.create test

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: src/app/endpoint/message.rs [hunk 1]
```diff
@@ -38,7 +38,7 @@ impl State {
 }
 
 impl State {
-    pub(crate) async fn create(&self, inreq: CreateRequest) -> Result<impl Publish, SvcError> {
+    pub(crate) async fn create(&self, inreq: CreateRequest) -> Result<OutgoingEnvelope, SvcError> {
```
Comment (manifest) [inline src/app/endpoint/message.rs]: We can't use `OutgoingEnvelope` as return value for any endpoint at the moment. There could be more than one object. In this particular case here will be a response and an event.

Comment (manifest) [reply src/app/endpoint/message.rs]: We need to pick the right return value for all our endpoints first.

Comment (feymartynov) [reply src/app/endpoint/message.rs]: Didn't get it.
`req.into_envelope().map_err(Into::into)` returns `OutgoingEnvelope` which implements `Publish`.
What's the problem with specifying concrete type insteand of trait here?

**Comment by feymartynov** [comment]
Closed in favor of #52.

**Pull request closed without merging**